ingestor/service.go

```diff
@@ -343,6 +348,12 @@ func (s *Service) HandleTransfer(w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
+	if !s.isOpen {
```

Comment:
I'd use the `DisableWrites()` which will shutdown the HTTP server altogether and close the `Store`.  That will prevent new writes from all paths.  

`isOpen` actually creates a race condition because `HandleTransfer` reads it and `Close` writes it in separate goroutines.  If you add a mutex to protect it, it creates lock contentions and slows write throughput if not done carefully.

Reply:
DisableWrites() function I only see close on store and metrics service. How does it shutdown http server?

Reply:
My mistake. Looks like the http shutdown is done in `main` just before `DisableWrites` is called.   We could either refactor the Service to include the http server so that `DisableWrites` could call it or use `isOpen` approach.  I'd say use `isOpen` but switch it to use atomics vs member var w/ a mutex.  I'd suggest going with the atomics for now (even though it's still racy), and then we can refactor the server/service separately and switch to that later.

Reply:
Reverted to using annotations based invocation on the ingestor pod. Hope this mitigates the denial of attack concerns.